generator/openapi/src/main/resources/templates/expediagroup-sdk/client/client.mustache

```diff
@@ -40,7 +40,7 @@ import {{import}}
         val response = httpClient.request {
             method = HttpMethod.parse("{{httpMethod}}")
             url("{{#removeLeadingSlash}}{{path}}{{/removeLeadingSlash}}"{{#pathParams}}.replace("{" + "{{baseName}}" + "}", {{#isContainer}}{{paramName}}.joinToString(","){{/isContainer}}{{^isContainer}}"${{{paramName}}}"{{/isContainer}}){{/pathParams}})
-            appendHeaders(transactionId)
+            appendHeaders(UUID.randomUUID())
```

Comment:
There is no need to send the value from the templates now, just set it inside the method.

Reply:
Done ✅